qlib/contrib/evaluate.py

```diff
@@ -238,8 +238,8 @@ def backtest_daily(
     portfolio_metric_dict, indicator_dict = backtest_func(
         start_time=start_time,
         end_time=end_time,
-        strategy=strategy,
-        executor=executor,
+        strategy_config=strategy,
```

Comment:
If you want to change this name,

the example in ` docs/component/strategy.rst ` should also be changed. 
But I think previous naming (i.e. `strategy`  or `executor`) is also OK. Because they could be neither config  or object